packages/flutter_driver/lib/src/driver/driver.dart

```diff
@@ -61,25 +61,38 @@ enum TimelineStream {
 
 const List<TimelineStream> _defaultStreams = <TimelineStream>[TimelineStream.all];
 
+/// Multiplies the timeout values used when establishing a connection to the
+/// Flutter app under test and obtain an instance of [FlutterDriver].
+///
+/// This multiplier applies automatically when using the default implementation
+/// of the [vmServiceConnectFunction].
+///
+/// See also:
+///
+///  * [FlutterDriver.timeoutMultiplier], which multiplies all command timeouts by this number.
+double connectionTimeoutMultiplier = _kDefaultTimeoutMultiplier;
+
+const double _kDefaultTimeoutMultiplier = 1.0;
+
 /// Default timeout for short-running RPCs.
-const Duration _kShortTimeout = Duration(seconds: 5);
+Duration _kShortTimeout(double multiplier) => const Duration(seconds: 5) * multiplier;
```

Comment:
Shouldn't these now be named `_shortTimeout`, etc.?  They're no longer const variables, and because of their names, I would be surprised to find that these took arguments.

Reply:
Good point! Which is also why I don't like the "k" prefixes and supported the move from `SCREAMING_CAPS_CONSTANTS` to `lowerCaseConstants` :)